src/utils/formatPageInfo/index.ts

```diff
@@ -5,13 +5,55 @@ import { formatDBTimestampToObject } from "./timestampFormatter";
 import type { SignatureData, Witness, SignatureStatus } from "../../types";
 import * as nameResolver from "../../name_resolver";
 
+type MerkleProofStatus = "VALID" | "INVALID" | "DOMAIN_SNAPSHOT";
+type EtherscanResult = "true" | "false" | "Transaction hash not found" | string;
+
+// let errMsg
+// if (etherScanResult === "Transaction hash not found") {
+//   errMsg = "Transaction hash not found"
+// } else if (etherScanResult.includes("ENETUNREACH")) {
+//   errMsg = "Server is unreachable"
+// } else {
+//   errMsg = "Online lookup failed"
+// }
+// result.etherscan_error_message = errMsg
+export type EtherscanErrorMessage =
+  | "Transaction hash not found"
+  | "Server is unreachable"
+  | "Online lookup failed"
+  | undefined;
+
+// result.extra = {
+//   domain_snapshot_genesis_hash: witnessData.domain_snapshot_genesis_hash,
+//   merkle_root: witnessData.merkle_root,
+//   witness_event_verification_hash:
+//     witnessData.witness_event_verification_hash,
+// }
+export type WitnessResultExtra = {
+  domain_snapshot_genesis_hash: string;
+  merkle_root: string;
+  witness_event_verification_hash: string;
+} | null;
+
+export type WitnessResultProps = {
+  actual_witness_event_verification_hash: string;
+  doVerifyMerkleProof: boolean;
+  etherscan_error_message: EtherscanErrorMessage;
+  etherscan_result: EtherscanResult;
```

Comment:
I think this is unused? I can't find a code that tells users to verify manually if this is "false".

Reply:
fair enough, will remove